BraveRewardsUI/Localized Strings/Strings.swift

```diff
@@ -41,6 +41,7 @@ internal extension Strings {
   static let SettingsTitle = NSLocalizedString("BraveRewardsSettingsTitle", bundle: Bundle.RewardsUI, value: "Settings", comment: "")
   static let Tips = NSLocalizedString("BraveRewardsTips", bundle: Bundle.RewardsUI, value: "Tips", comment: "")
   static let SettingsAutoContributeTitle = NSLocalizedString("BraveRewardsSettingsAutoContributeTitle", bundle: Bundle.RewardsUI, value: "Auto-Contribute", comment: "")
+  static let SettingsAutoContributeUpToValue = NSLocalizedString("BraveRewardsSettingsAutoContributeTitle", bundle: Bundle.RewardsUI, value: "Up to %@", comment: "The maximum amount of BAT to auto-contribute. E.g. 'Up to 20.0 BAT'")
```

Comment:
`BraveRewardsSettingsAutoContributeTitle` forgot to change this when copy pasting

Reply:
Good catch :)